### PR TITLE
Update DQM onlinebeammonitor

### LINKED ISSUE #34717: Update to DQM onlinebeammonitor
In order to properly monitor the updates of the BeamSpotOnline tags at lumisection level the configuration of the client
[onlinebeammonitor_dqm_sourceclient-live_cfg.py](https://github.com/cms-sw/cmssw/blob/master/DQM/Integration/python/clients/onlinebeammonitor_dqm_sourceclient-live_cfg.py) needs to be updated to include the `refreshTime` parameter.
This change only affects the **monitoring** of the BeamSpotOnline updates on the DQM side. 

The fix is rather simple (just adding a `toGet` to customize the process.GlobalTag is enough) and it has been tested already in the HLT menu live at P5 (see http://cmsonline.cern.ch/cms-elog/1121707).
I will prepare PRs for 11_3, 12_0 and 12_1 by today. 

@perrotta @qliphy @silviodonato This is just to let you know that it would be nice to include this in the new build for CRUZET (not sure whether is this already planned or if the issues with HCAL GPU code still do not have a solution).

FYI
@gennai @ggovi 
@amassiro @boudoul 
@cms-sw/dqm-l2

### PR DESCRIPTION
**Pull request by francescobrivio**
#### PR description:
Fixes #34717.
Added a VPSet to `process.GlobalTag` in  `onlinebeammonitor_dqm_sourceclient-live_cfg.py` DQM client:
```
process.GlobalTag.toGet = cms.VPSet(
  cms.PSet(
    record = cms.string("BeamSpotOnlineLegacyObjectsRcd"),
    refreshTime = cms.uint64(1)
  ),
  cms.PSet(
    record = cms.string("BeamSpotOnlineHLTObjectsRcd"),
    refreshTime = cms.uint64(1)
  )
)
```
In this way the information about BeamSpotOnline is fetched every lumisection and the monitoring
actually reflects the behaviour of any other consumer of these tags (most notably the HLT).
Note that only the record name is needed in the toGet, this makes the client configuration resilient for
future changes of the tag names themselves.

#### PR validation:
Compiles.
Also tested online and deployed the same configuration change in the HLT cosmics menu running in P5 (see http://cmsonline.cern.ch/cms-elog/1121707).

#### Backport
Backports to 12_0_X (needed for CRAFT and BeamTest) and 11_3_X (for CRUZET) will be previded shortly.

### CODE REVIEW

**Comment by francescobrivio** [comment]
@cmsbuild please test

File: DQM/Integration/python/clients/onlinebeammonitor_dqm_sourceclient-live_cfg.py [hunk 1]
```diff
@@ -144,7 +144,16 @@
 # Condition for lxplus: change and possibly customise the GT
 #from Configuration.AlCa.GlobalTag import GlobalTag as gtCustomise
 #process.GlobalTag = gtCustomise(process.GlobalTag, 'auto:run2_data', '')
-
+process.GlobalTag.toGet = cms.VPSet(
```
Comment (mmusich) [inline DQM/Integration/python/clients/onlinebeammonitor_dqm_sourceclient-live_cfg.py:147]: I guess one line of comment explaining why this is needed would make it more robust against future accidental 'clean-up' of `toGet` statements.

Comment (francescobrivio) [reply DQM/Integration/python/clients/onlinebeammonitor_dqm_sourceclient-live_cfg.py:147]: good point, I will add a brief description!

**Comment by francescobrivio** [comment]
@cmsbuild please test

**Comment by francescobrivio** [comment]
From the logs I see for example:
```
02-Aug-2021 18:03:58 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/user/cmsbuild/store/relval/CMSSW_12_0_0_pre4/RelValSingleElectronPt10/GEN-SIM/120X_mcRun3_2021_realistic_v2-v1/00000/b777b51b-e7ee-4b6f-8955-32def06276e7.root

A fatal system signal has occurred: external termination request
The following is the call stack containing the origin of the signal.

A fatal system signal has occurred: external termination request
The following is the call stack containing the origin of the signal.

Waiting for stacktrace completion failed: timed out waiting for GDB to complete.

Current Modules:

Module: none (crashed)

A fatal system signal has occurred: external termination request
```

@qliphy @perrotta I guess this is a temporary glitch accessing the input files?
Should I re-trigger the tests right away?

**Comment by francescobrivio** [comment]
BTW this also failed in 12_0_X (https://github.com/cms-sw/cmssw/pull/34726#issuecomment-891300451) but not in 11_3_X (https://github.com/cms-sw/cmssw/pull/34721#issuecomment-891205808).

**Comment by francescobrivio** [comment]
@cmsbuild please test

**Comment by perrotta** [comment]
> 
> @qliphy @perrotta I guess this is a temporary glitch accessing the input files?
> Should I re-trigger the tests right away?

Those RelVal Input tests are failing quite often, also in unrelated PR.
Thank you for having re-triggered the tests, would they fail again just ignore them.
(This is quite annoying: would a real error show up in those tests, I bet nobody will take care of...)

**Comment by francescobrivio** [comment]
Failures are: 
 - wf 8.0 --> usual failure
 - wf 10224.0 --> only MessageLogger
 - wf 140.53 --> this is Run2011 HI, I have no idea why there are so many differences/failures here, should be completely unrelated to this PR

**Comment by perrotta** [comment]
> Failures are:
> 
> * wf 8.0 --> usual failure
> * wf 10224.0 --> only MessageLogger
> * wf 140.53 --> this is Run2011 HI, I have no idea why there are so many differences/failures here, should be completely unrelated to this PR

@francescobrivio see https://github.com/cms-sw/cmssw/pull/34719#issuecomment-891776396:
"The workflows 140.53 have different files in step1_dasquery.log than the ones found in the baseline. You may want to check and retrigger the tests if necessary. You can check it in the "files" directory in the results of the comparisons"

**Comment by mmusich** [comment]
From https://github.com/cms-sw/cmssw/pull/34719#issuecomment-891776396

> The workflows 140.53 have different files in step1_dasquery.log than the ones found in the baseline. You may want to check and retrigger the tests if necessary. You can check it in the "files" directory in the results of the comparisons

**Comment by francescobrivio** [comment]
thanks @perrotta and @mmusich I had missed that message!

**Comment by francescobrivio** [comment]
FYI the DQM team has also tested the 11_3_X backport live in their playback system in P5 and found no problems.

**Comment by jfernan2** [comment]
+1
Backport #34721 tested in P5 DQM playback sucessfully

**Comment by perrotta** [comment]
+1

**Pull request merged**